README.md

```diff
@@ -207,10 +207,20 @@ gulp test-coverage
 gulp view-coverage
 ```
 
-For end-to-end testing, edit the example file `./integrationExamples/gpt/pbjs_example_gpt.html`:
+For end-to-end testing:
 
-1. Change `{id}` values appropriately to set up ad units and bidders
-2. Set the path to Prebid.js in your example file as shown below (see `pbs.src`).
+```bash
+gulp e2e-test --host=test.localhost
+```
+
+To run these tests, the following items are required:
+- setup an alias of localhost in your `hosts` file (eg `127.0.0.1  test.localhost`); note - you can use any alias.  Use this alias in the command-line argument above.
+- access to [BrowserStack](https://www.browserstack.com/) account.  Assign the following variables in your bash_profile:
```

Comment:
I don't see how we can require most users to have access to a browserstack account. I don't think even most of us PBJS regulars have access to the account.

I would suggest that reference to this test be added to https://github.com/prebid/Prebid.js/blob/master/RELEASE_SCHEDULE.md instead of this file.

Regular community members should have the previous end-to-end option IMO.

Reply:
@bretg This is not a requirement. In our README we have mentioned gulp commands to do all the tasks. Since we updated our end to end test architecture, we are documenting the new commands here. Maybe we can add the language stating that it is optional ?

For prebid users, this is completely optional. It is not mandatory to run end to end tests.

Reply:
Ok - I added a note here about needing to be a prebid.org member to do these tests. Am satisfied with that.